ui/src/views/compute/DeployVM.vue

```diff
@@ -1285,6 +1287,9 @@ export default {
     }
   },
   methods: {
+    updateTemplateKey () {
+      this.templateKey += 1
```

Comment:
check if templateKey method receives an arg?

Reply:
Sorry @rohityadavcloud I don't understand why to check for an arg in this case, you mean to extend the method and increment only when an arg is passed?